src/lib/features/project/project-store.ts

```diff
@@ -579,32 +581,84 @@ class ProjectStore implements IProjectStore {
     }
 
     async getApplicationsByProject(
-        projectId: string,
-    ): Promise<IProjectApplication[]> {
+        params: IProjectApplicationsSearchParams,
+    ): Promise<IProjectApplications> {
+        const { project, limit, sortOrder, sortBy, searchParams, offset } =
+            params;
+        const validatedSortOrder =
```

Comment:
Maybe this could be done in the schema level and reject requests with 400 bad request the contents are not 'asc' or 'desc' ?

Reply:
I want to be absolutely sure also on store level, because this can be called from codebase and is potential SQL injection hole.